src/packages/shared-utils/package-check.ts

```diff
@@ -11,13 +12,8 @@ const secondClockStatuses = [
   SEATOOL_STATUS.PENDING_CONCURRENCE,
 ];
 
-const checkAuthority = (
-  authority: Authority | null,
-  validAuthorities: Authority[],
-) =>
-  !authority
-    ? false
-    : validAuthorities.includes(authority.toLowerCase() as Authority);
```

Comment:
Removing the `.toLowerCase` method changes the test arguments below. So `medicaid spa` turns to `Medicaid SPA`

Reply:
I believe the API also returns `authority` normally i.e. not lowercased